analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UseReturnStatement.vb

```diff
@@ -124,6 +139,20 @@ Public Class DoesNotApplyOn
     Private Sub WithExplicitArgumentName(ArgumentName As Integer)
     End Sub
 
+    ' https://github.com/SonarSource/sonar-dotnet/issues/4347
+    Public Function OtherType() As OtherType    ' Noncompliant FP
+        Dim Ret As OtherType                    ' Noncompliant FP
+    End Function
+
+    Public Function NamespaceName() As NamespaceName.Something  'Noncompliant FP
+        Dim Ret As NamespaceName.Something                      'Noncompliant FP
+    End Function
+
+    Public Sub SomeEvent() Handles fSource.SomeEvent
+        Dim S As Source
+        AddHandler S.SomeEvent, AddressOf SomeEvent
```

Comment:
I don't get it - is this part of the repro?

Reply:
No, this is just another unrelated syntax that came to my mind while working on it and I wanted to be sure we don't have another FP there.